cmd/flags.go

```diff
@@ -164,6 +166,18 @@ func urlFlag(cmd *cobra.Command) *cobra.Command {
 	return cmd
 }
 
+func strategyFlag(cmd *cobra.Command) *cobra.Command {
```

Comment:
Going to merge as is, but I think this would benefit from a slightly better name in an ideal world. something like `txSizeFlags`.

Reply:
Mainly cause this has me thinking I should move the `Strategy` field off of `Path` and have it be passed in via a command line flag where applicable. https://github.com/iqlusioninc/relayer/issues/262